lib/srv/desktop/rdp/rdpclient/src/rdpdr/filesystem.rs

```diff
@@ -719,7 +825,7 @@ impl FilesystemBackend {
         file: &FileCacheObject,
     ) -> PduResult<()> {
         let file_fso_size: i64 = cast_length!(
-            "FilesystemBackend::make_client_drive_query_information_response",
+            "FilesystemBackend::send_file_standard_info",
```

Comment:
I wonder if something like https://stackoverflow.com/a/40234666/914991 would be worthwhile for a lot of these areas where you're manually writing the name of the enclosing function. These are prone to copy-pasta errors.

Reply:
Good call, I'll add that to a PR on the top of the stack.